packages/vue-vanilla/src/cells/TextCell.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <input
+    :id="control.id + '-input'"
+    :class="styles.control.input"
+    :value="control.data"
+    :disabled="!control.enabled"
+    :autofocus="appliedOptions.focus"
+    :placeholder="appliedOptions.placeholder"
+    @change="onChange"
+    @focus="isFocused = true"
+    @blur="isFocused = false"
+  />
+</template>
+
+<script setup lang="ts">
+import {
+  ControlElement,
+  isStringControl,
+  type RankedTester,
+  rankWith,
+} from '@jsonforms/core';
+import { rendererProps, useJsonFormsControl } from '@jsonforms/vue';
+import { useVanillaControl } from '../util';
+
+const props = defineProps(rendererProps<ControlElement>());
+
+const input = useVanillaControl(
+  useJsonFormsControl(props),
+  (target) => target.value || undefined
+);
```

Comment:
We should implement and use cell specific variants of the `useJsonFormsControl` and maybe `useVanillaControl` bindings, like we do in React.

By reusing the one for the `renderers`, we will calculate a lot of unnecessary props here (e.g. label) which are then never used.

Reply:
@sdirix is the implementation in the current commit as you imagine it? I have used the useVanillaCell() and useJsonFormsCell() functions at the cells, whereby the useVanillaCell() is actually only a copy of the useVanillaControl(), so far correct?